src/platform/user/authentication/utilities.js

```diff
@@ -285,18 +286,21 @@ export function redirect(redirectUrl, clickedEvent, type = '') {
   window.location = redirectUrl;
 }
 
+export async function mockLogin({ clickedEvent = AUTH_EVENTS.MOCK_LOGIN }) {
+  const url = `${MOCK_LOGIN_URL}`;
```

Comment:
Generate the url using the `createOAuthRequest`, it should still have the other OAuth query params besides the `type`.

Reply:
Sounds good, will make these adjustments.